apps/parsley/src/components/ProjectFiltersModal/index.tsx

```diff
@@ -35,13 +35,13 @@ const ProjectFiltersModal: React.FC<ProjectFiltersModalProps> = ({
 
   const { task } = useTaskQuery({ buildID, execution, logType, taskID });
   const { versionMetadata } = task ?? {};
-  const { projectIdentifier = "" } = versionMetadata ?? {};
+  const { projectMetadata } = versionMetadata ?? {};
 
   const { data } = useQuery<ProjectFiltersQuery, ProjectFiltersQueryVariables>(
     PROJECT_FILTERS,
     {
-      skip: !projectIdentifier,
-      variables: { projectIdentifier },
+      skip: !projectMetadata?.id,
+      variables: { projectIdentifier: projectMetadata?.id ?? "" },
```

Comment:
Can we update the query to take `projectId` so this looks more natural? (not the underlying query itself, but the wrapper definition we have for the query in the GraphQL file)